src/endpoint.cpp

```diff
@@ -387,9 +389,9 @@ bool Endpoint::accept_msg(int target_sysid, int target_compid, uint8_t src_sysid
     if (has_sys_comp_id(src_sysid, src_compid))
         return false;
 
-    if (msg_id != UINT32_MAX &&
-        _message_filter.size() > 0 &&
-        std::find(_message_filter.begin(), _message_filter.end(), msg_id) == _message_filter.end()) {
```

Comment:
Is there anyway to stop clang-format to change those (other than with a comment on top, that doesn't scale)? Having conditions in single lines sometimes make it easier to read.... Not always, but in this case I very much prefer the way it was, even if it becomes a long line.

Reply:
I'll take a look at it again, but I guess, that it won't be possible. You can always enclose the lines with `// clang-format off` and `// clang-format on`, but that wasn't the question.

Reply:
for some places, like the tables, the comment make sense. They will always be tables. For places in the middle of the code, those comments do more harm than good: we either a) find a config we like or b) accept the fact it's better to be consistent than to have everything perfect or c) do not enforce it in some places